tests/test_gtfs_objects.py

```diff
@@ -0,0 +1,251 @@
+import datetime
+import os
+import pytest
+
+import numpy as np
+import pandas as pd
+
+from fasttrips import Route
+from fasttrips import Stop
+from fasttrips import Transfer
+from fasttrips import Trip
+
+HOME_DIR = os.path.join(os.getcwd(), "fasttrips", "Examples", )
+NETWORK_HOME_DIR = os.path.join(HOME_DIR, 'networks')
+
+
+@pytest.fixture(scope='module', params=['simple'])
+def scenario(request):
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def network_dir(scenario):
+    yield os.path.join(NETWORK_HOME_DIR, scenario)
+
+
+@pytest.mark.skip(reason='Need to upgrade to Partridge')
+def test_stops_load(network_dir, gtfs_feed):
```

Comment:
add description

Reply:
Cleaned up test_gtfs_objects to be compatible with Partridge. Added docstrings.